components/CandidateGuid/CandidateGuid.tsx

```diff
@@ -0,0 +1,20 @@
+﻿import textStyles from "../CandidateGuid/CandidateGuid.module.scss";
+import React from "react";
+
+interface CandidateProps {
+    guid: any;
+}
+
+
+export function CandidateGuid(props: CandidateProps): JSX.Element {
+    const baseUrl = `https://testswitch-api-staging.herokuapp.com/sessions`;
```

Comment:
The idea is that this is the `URL` that we could send to the student for them to start their tests.
So (if hardcoded) it final URL should be ``https://testswitch-candidate.herokuapp.com?token=${guid}`` 

In an even more idea world, this could be changed via a environment variable so that it worked correctly on both staging and production! :)

Reply:
have hardcoded for now, might come back in a bit and look at adding as env 👍